rust-runtime/aws-smithy-http-server/examples/pokemon_service/tests/simple_integration_test.rs

```diff
@@ -10,15 +10,42 @@
 use std::time::Duration;
 
 use crate::helpers::{client, PokemonService};
+use async_stream::stream;
```

Comment:
Be aware the this file is currently symlinked inside the Python folder. You will need to copy the original implementation there since event streams are not supported yet.